packages/docs/src/routes/qwikcity/endpoint/non-200.mdx

```diff
@@ -65,3 +67,39 @@ export default component$(() => {
   );
 });
 ```
+
+## 3xx - Redirect
+
+Let's say that a user does a request to an invalid `skuId` such as `https://example.com/product/999/details`. In this case, we would like to return a 404 HTTP status code and render a 404 page. The place where we determine if the request is valid or not is the data endpoint by looking into the database. Even if the response is non-200, the component still gets a chance to render a page (Except in the redirect case.)
```

Comment:
I think the text sholud talk about 303 not 404. Looks like copy-pasta.

Reply:
Fixed!